extern/wine/winuser.rh

```diff
@@ -0,0 +1,1401 @@
+/*
```

Comment:
this can be placed into `libs/common` as well if it's used globally

Reply:
Done.